pkg/bazel/analysis_v2.pb.go

```diff
@@ -0,0 +1,1331 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Why did this get checked in?

Reply:
same reason as flags.pb.go next to it - editor tooling is not working for codegen in bazel-out. I'll remove this from the PR for now, it's not load-bearing and also it's missing an up-to-date test